pkg/controller/direct/compute/targettcpproxy/targettcpproxy_controller.go

```diff
@@ -260,11 +262,16 @@ func (a *targetTCPProxyAdapter) Update(ctx context.Context, updateOp *directbase
 	}
 	region := parent.Region
 
+	// Regional API does not support Update
+	if region != "global" {
```

Comment:
This could be error prone and hard to maintain. Can we separate 'global' from region or use a different field?

Reply:
Can we separate 'global' from 'region'

Do you mean separate them into two resources? I think the reason we decided to support them in one CRD is because it's align with other regional resources, and easier for user adoption. In addition, since the resource will be used as a reference in CFR, we do not need to add another ref resource type.

or use a different field

Not sure why that would help? The shared and global mutable fields are "service" and "proxyHead".

Reply:
Done, use `location` instead.